### PR TITLE
Added SDG variable override config to highlight tile

### PR DESCRIPTION
**Pull request by dwnoble**
Before:
highlight-before

After:
highlight-after

### CODE REVIEW

File: static/js/components/tiles/highlight_tile.tsx [hunk 1]
```diff
@@ -134,6 +196,7 @@ const fetchData = (props: HighlightTilePropType): Promise<HighlightData> => {
       paramsSerializer: stringifyFn,
     })
     .then((resp) => {
+      console.log("resp.data=", resp.data);
```
Comment (juliawu) [inline static/js/components/tiles/highlight_tile.tsx:199]: Remove?

**Review by juliawu — APPROVED** [review]
Nice!

**Review by n-h-diaz — APPROVED** [review]
thank you!!

File: static/js/components/tiles/highlight_tile.tsx (hunk 1 above)
Comment (dwnoble) [reply static/js/components/tiles/highlight_tile.tsx]: Whoops, removed

**Pull request merged**